arches/app/src/components/ControlledListManager/ControlledListInventory.vue

```diff
@@ -0,0 +1,196 @@
+<script setup lang="ts">
+import arches from "arches";
+import Cookies from "js-cookie";
+import { ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Splitter from "primevue/splitter";
+import SplitterPanel from "primevue/splitterpanel";
+import { useToast } from "primevue/usetoast";
+
+import ControlledListTable from "@/components/ControlledListManager/ControlledListTable.vue";
+import SidepanelDataView from "@/components/ControlledListManager/SidepanelDataView.vue";
+import SpinnerIcon from "@/components/SpinnerIcon.vue";
+
+import type { Ref } from "vue";
+import type {
+    ControlledList,
+    LanguageMap,
+} from "@/types/ControlledListManager.d";
+
+const props: {
+    displayedList: ControlledList | null;
+    languageMap: LanguageMap | null;
+    setDisplayedList: (list: ControlledList | null) => void;
+    setLanguageMap: (map: LanguageMap) => void;
+    setEditing: (val: boolean) => void;
+} = defineProps([
+    "displayedList",
+    "languageMap",
+    "setDisplayedList",
+    "setLanguageMap",
+    "setEditing",
+]);
+
+const items: Ref<ControlledList[]> = ref([]);
+const toast = useToast();
+const { $gettext, $ngettext } = useGettext();
+const lightGray = "#f4f4f4";
+
+const fetchLists = async () => {
+    let errorText;
+    try {
+        const response = await fetch(arches.urls.controlled_lists);
+        if (!response.ok) {
+            errorText = response.statusText;
+            const body = await response.json();
+            errorText = body.message;
```

Comment:
double `errorText` assignment?

Reply:
If `await response.json()` throws, e.g. because the backend view tripped on something else and didn't actually return JSON, we can at least get `statusText` from the generic 500 response and show "Internal Server Error" to distinguish "server responded" from "you don't have an internet connection"

Reply:
Otherwise, I'm using body.message, which might have a localized field validation msg